src/schema/types.ts

```diff
@@ -147,3 +148,24 @@ export type DataTrackedAttrs = {
   userID: string
   createdAt: number
 }
+
+export type SectionCategory = {
+  _id: string
+  name: string
```

Comment:
I don't think this is needed, the "titles" should be enough.

Reply:
Removed.